navigation/IdeaNavigator.js

```diff
@@ -1,24 +1,37 @@
-import { createStackNavigator } from 'react-navigation-stack';
-import { createAppContainer, createSwitchNavigator } from 'react-navigation';
-
+import React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
 import { ExplorePage } from '../containers/Ideas/ExplorePage';
 import { Idea } from '../containers/Ideas/Idea';
 import { IdeaCreate } from '../containers/Ideas/IdeaCreate';
 import { IdeaProject } from '../containers/Ideas/IdeaProject';
 import { LoginScreen } from '../containers/Auth/LoginScreen';
 import { StartUpScreen } from '../containers/Auth/StartUpScreen';
+import { useAuthorization } from '../containers/Auth/AuthProvider.js';
 
-const IdeaNavigator = createStackNavigator({
-  ExplorePage: ExplorePage,
-  IdeaProject: IdeaProject,
-  IdeaDetail: Idea,
-  IdeaCreate: IdeaCreate
-});
-
-const MainNavigator = createSwitchNavigator({
-  StartUp: StartUpScreen,
-  Auth: LoginScreen,
-  Ideas: IdeaNavigator
-});
+const Stack = createStackNavigator();
 
-export default createAppContainer(MainNavigator);
+export const IdeaNavigator = (props) => {
+  const { loading, token } = useAuthorization();
+  return (
+    <NavigationContainer>
+      <Stack.Navigator
+        initialRouteName="StartUp"
+        screenOptions={{ gestureEnabled: false, headerShown: false }}
```

Comment:
this will hide the header bar with the back arrow and everything, which I think is what we want?